ScoutSuite/providers/aws/resources/organizations/accounts.py

```diff
@@ -0,0 +1,24 @@
+from ScoutSuite.providers.aws.resources.base import AWSResources
+
+
+class Accounts(AWSResources):
+    async def fetch_all(self):
+        raw_accounts = await self.facade.organizations.get_accounts()
+        print("\nAccounts in this Organization: \n")
```

Comment:
Looks like there are a bunch of extra print statements in here to take out before we merge